### ISSUE TITLE
source.organizeImports no longer applied on saving

### ISSUE THREAD

**Issue by rchl** [microsoft/pyright#926]
**Describe the bug**

VSCode and some other client implementations support running code actions on save. This is handled by triggering `textDocument/codeAction` request with `context.only` set to specific code action to receive (in this case that would be `source.organizeImports`). That has worked before https://github.com/microsoft/pyright/commit/2d0e3eb4c1d11c505bf07d0ccd881f71e06b5da7 but no longer does. That change has fixed a bug where "organize imports" code action would be returned for all locations in the view and while that is a good change, it should still be returned when asking with `context.only` set to `source.organizeImports`.

[1] https://microsoft.github.io/language-server-protocol/specification#textDocument_codeAction

**To Reproduce**

1. Configure editor (VSCode) with:
```js
  "editor.codeActionsOnSave": {
    "source.organizeImports": true
  },
```
2. Save the document with imports.

**Expected behavior**

Imports are organized

**VS Code extension or command-line**

Using pylance extension but trying to save the file with the mentioned configuration in VSCode currently seems to gets stuck at this:
![Screenshot 2020-08-09 at 17 09 33](https://user-images.githubusercontent.com/153197/89735485-1a889e80-da63-11ea-8fc5-e5514ccd3258.png)

But this issue is more about pure server than pylance or vscode.

**Comment by erictraut** [comment]
This was an intentional change in Pylance.

From the release notes in the 2020.7.0 release:
> Import organization has been disabled to prevent conflicts with the Python extension's import sorting.

The problem is that the Python extension already provides support for import sort ordering. Pylance was conflicting with this functionality. If you want auto-sorting of imports, you can install the standard "isort" tool and configure it by following the instructions in this article: https://medium.com/@cereblanco/setup-black-and-isort-in-vscode-514804590bf9.

**Comment by rchl** [comment]
Disclaimer: I'm not a VSCode user and I'm looking at this issue from the perspective of the language server user, so features provided by vscode extension are not really relevant from that point of view.

But I can understand if you think that it's not important for you to support it and I'm fine with that.

But it could have been solved by providing a setting so that it's possible to satisfy both use-cases.

**Comment by erictraut** [comment]
Yeah, import sort ordering is really more of a stylistic / linting feature and is outside the scope of Pyright, which focuses on type checking, not code style issues.

**Comment by rchl** [comment]
I believe you are still supporting it though. It's just that it's a server command that has to be explicitly invoked.

**Comment by erictraut** [comment]
Yes, I initially added it when Pyright didn't run well with the VS Code Python extension. Now that Pylance is released and is designed to work with the Python extension, it probably makes sense to just delete this functionality from the Pyright source base.
